src/frontend/src/components/public/PIAFormTabs/review/viewProgramArea.tsx

```diff
@@ -43,7 +43,7 @@ const ViewProgramAreaReview = (props: IProgramAreaReviewProps) => {
           pia={pia}
           reviewSection={pia?.review?.programArea?.review}
           editReviewNote={editReviewNote}
-          printPreview
+          printPreview={false}
```

Comment:
I would rather delete the this line to avoid confusion altogether at both the instances.

Reply:
done